cmake/modules/gluecodium/gluecodium/Generate.cmake

```diff
@@ -158,7 +159,7 @@ function(apigen_generate)
       DART_LIBRARY_NAME
       DART_FUNCTION_LOOKUP_ERROR_MESSAGE
       DART_NAMERULES)
-  set(multiValueArgs LIME_SOURCES FRANCA_SOURCES WERROR)
+  set(multiValueArgs LIME_SOURCES FRANCA_SOURCES WERROR TAG)
```

Comment:
Call it TAGS, plural, as other multi value args

Reply:
Done